packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -218,13 +219,23 @@ function parseAmount(amount, mapper) {
   return value;
 }
 
-function parseAmountFields(trans, splitMode, flipAmount) {
+function parseAmountFields(trans, splitMode, flipAmount, multiplierAmount) {
+  const multiplier = (() => {
+    if (multiplierAmount === '') {
+      return parseFloat('1');
+    } else {
+      return parseFloat(multiplierAmount);
+    }
+  })();
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  const multiplier = parseFloat(multiplierAmount) || 1.0;
```